controllers/apply/under10k/fields/project-dates-next.js

```diff
@@ -0,0 +1,249 @@
+            .format('DD MM YYYY');
+
+        function schema() {
+            /**
+             * When projectStartDateCheck is asap
+             * we don't show the project start date question
+             * and instead pre-fill it with the current date
+             * at the point of submission (see forSalesforce())
+             */
+            return Joi.when('projectStartDateCheck', {
+                is: 'asap',
+                then: Joi.any().strip(),
+                otherwise: Joi.dateParts()
+                    .minDate(minDate.format('YYYY-MM-DD'))
+                    .required(),
+            });
+        }
+
+        return new DateField({
+            locale: locale,
+            name: 'projectStartDate',
+            label: localise({
+                en: `Tell us when you'd like to get the money if you're awarded funding?`,
+                cy: `@TODO: i18n`,
+            }),
+            explanation: localise({
+                en: oneLine`Don't worry, this can be an estimate. 
+                But most projects must usually start on or after
+                <strong>${minDateExample}</strong>.`,
+                cy: oneLine`Peidiwch â phoeni, gall hwn fod yn amcangyfrif. 
+                Ond fel rheol mae'n rhaid i'r mwyafrif o brosiectau ddechrau ar 
+                neu ar ôl <strong>${minDateExample}</strong>.`,
+            }),
+            settings: {
+                minYear: minDate.format('YYYY'),
+            },
+            schema: schema(),
+            messages: [
+                {
+                    type: 'base',
+                    message: localise({
+                        en: `Enter a project start date`,
+                        cy: `Cofnodwch ddyddiad dechrau i’ch prosiect`,
+                    }),
+                },
+                {
+                    type: 'dateParts.minDate',
+                    message: localise({
+                        en: oneLine`Date you start the project must be on
+                            or after ${minDateExample}`,
+                        cy: oneLine`Mae’n rhaid i ddyddiad dechrau eich
+                            prosiect fod ar neu ar ôl ${minDateExample}`,
+                    }),
+                },
+            ],
+        });
+    },
+    fieldProjectEndDate(locale, data = {}) {
+        const localise = get(locale);
+
+        const projectCountry = get('projectCountry')(data);
+
+        function getMaxDurationMonths() {
+            if (projectCountry === 'england') {
+                return 6;
+            } else {
+                return 12;
+            }
+        }
+
+        function explanation() {
+            if (projectCountry === 'england') {
+                return localise({
+                    en: oneLine`Given the COVID-19 emergency, you can have up to
+                        ${getMaxDurationMonths()} months after award to spend the money.
+                        It can even be as short as just one day.`,
```

Comment:
The final line here doesn't make much sense in this new context, I think we can improve this a bit.